.github/workflows/main.yml

```diff
@@ -77,12 +75,10 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: ${{ matrix.platform.python-architecture }}
       - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
```

Comment:
Why is this `master` while other jobs use `stable` below?

Reply:
So, this Github action is a little bit funny because you can specify the version in the `@`. `@stable` installs Rust stable, `@1.57.1` installs 1.57.1 etc. But if you want to have the version as an input (like from the matrix we have), the author suggests using the `toolchain` field: https://github.com/dtolnay/rust-toolchain#inputs